docs/en/administrator-guide/config/fe_config.md

```diff
@@ -138,6 +138,12 @@ Default：1G
 
 Used to set the initial flow window size of the GRPC client channel, and also used to max message size.  When the result set is large, you may need to increase this value.
 
+### min_table_replication_num
```

Comment:
what's the difference between min_table_replication_num and min_replication_num_per_tablet? it seems same?

Reply:
Oh, I didn't notice that they are indeed the same thing...